macOS/LocalPackages/SubscriptionUI/Sources/SubscriptionUI/Preferences/PreferencesSubscriptionSettingsModelV1.swift

```diff
@@ -0,0 +1,339 @@
+//
+//  PreferencesSubscriptionSettingsModelV1.swift
```

Comment:
This has to temporarily exist in V1 and V2 variants given partially incompatible SubscriptionManager APIs. Will be removed and renamed post Auth V2 release.